### ISSUE TITLE
Feature request: send files selected in finder and send them to channel/user

### ISSUE THREAD

**Issue by nimbling** [yannickglt/alfred-slack#51]
Hi Yannickglt!

Thanks for making this integration.

I have a feature request: I'd like to select a few files in the finder and zip & send them to a channel/user.

Is this already possible using this workflow? Or is it a feasible thing to make?
Thanks for having a look!

**Comment by thespacedoctor** [comment]
Agreed this would be a super-useful feature. Thanks for developing this!
